app/code/Magento/Backend/Model/Url.php

```diff
@@ -166,7 +167,10 @@ protected function _isSecure()
         if ($this->hasData('secure_is_forced')) {
             return $this->getData('secure');
         }
-        return $this->_scopeConfig->isSetFlag('web/secure/use_in_adminhtml');
+        if ($this->_scopeConfig->isSetFlag('web/secure/use_in_adminhtml')) {
+            return true;
```

Comment:
> it also returns mostly true/false

omg. maybe just return true or false randomly then?

implementation must be changed so that `web/secure/use_in_adminhtml = 0` is still effective.

Reply:
@orlangur please stop commenting my issues please.. i am really angry on you because you are not wise enough to understand that there is a security bug in magento and web/secure/use_in_adminhtml = 0 is causing it.. Also your opinion that infinite redirects is not magento issue is crazy. You have told that you will leave this PR to the other developers, and still you are blocking the release..

Question to someone else: What needs to be done for web/secure/use_in_adminhtml = 0 to get obsolete in favor of fixing the security bug in magento?

Reply:
There is no "security bug" if Magento behaves exactly how you configured it. You cannot simply remove a part of functionality making `web/secure/use_in_adminhtml = 0` ineffective.

Reply:
You do not understand even written text that i do NOT want reply from YOU? Is Vlad Veselov only magento developer? If so, i am leaving magento forever..

Reply:
It does not work like that. During code review valuable feedback from *any* Community member should be incorporated.

Reply:
so stop spamming, and let other community members tell their review.. i believe that you have expressed enough your intelligence quotient at the thread..